src/garrysmod/gamemodes/zombiesurvival/entities/entities/prop_obj_sigil/shared.lua

```diff
@@ -54,3 +54,8 @@ function ENT:CanBeDamagedByTeam(teamid)
 
 	return teamid == TEAM_UNDEAD
 end
+
+-- Disable human collisions on sigil
+function ENT:ShouldNotCollide(ent)
+	return ent:IsPlayer() and ent:Team() == TEAM_HUMAN
+end
```

Comment:
Nitpick but there used to be a new line at the end of this file.

```suggestion
end

```